src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -134,6 +137,17 @@ function CreatePipelineRun(props) {
     })
   });
 
+  function handleToggleChange(event) {
+    const isPending = event.target.checked;
+    if (isPending) {
+      setState(state => ({
+        ...state,
+        pipelinePendingStatus: 'PipelineRunPending'
+      }));
+    } else {
+      setState(state => ({ ...state, pipelinePendingStatus: '' }));
+    }
```

Comment:
we can flip this inside out since the only piece changing is the status value:

```js
setState(state => ({
  ...state,
  pipelinePendingStatus: isPending ? 'PipelineRunPending' : ''
}));
```